app/javascript/channels/consumer.js

```diff
@@ -0,0 +1,6 @@
+// Action Cable provides the framework to deal with WebSockets in Rails.
+// You can generate new channels where WebSocket features live using the `bin/rails generate channel` command.
+
+import {createConsumer} from "@rails/actioncable";
+
+export default createConsumer("ws://localhost:3000/csc108/cable");
```

Comment:
Here it seems like you can use [action_cable_meta_tag](https://api.rubyonrails.org/classes/ActionCable/Helpers/ActionCableHelper.html#method-i-action_cable_meta_tag) to avoid hard-coding the URL.